pkgs/development/libraries/libopendmarc/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, libmilter }:
+
+stdenv.mkDerivation rec {
+  pname = "libopendmarc";
+  version = "1.3.3";
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ libmilter ];
+
+  postPatch = ''
+    substituteInPlace configure.ac --replace '	docs/Makefile' ""
+  '';
+
+  configureFlags = [
+    "--with-milter=${libmilter.out}"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "trusteddomainproject";
+    repo = "opendmarc";
+    rev = "rel-opendmarc-${builtins.replaceStrings [ "." ] [ "-" ] version}";
+    sha256 = "sha256-SQH85FLfVEEtYhR1+A1XxCDMiTjDgLQX6zifbLxCa5c=";
+  };
```

Comment:
Put the `src` attribute after `version`.

Reply:
I don't think I understand what you're asking. `version` is in line 5 and `src` in line 17?

Reply:
Sorry, I meant immediately after. So:

```
  version = "1.3.3";

  src = fetchFromGitHub {
    owner = "trusteddomainproject";
    repo = "opendmarc";
    rev = "rel-opendmarc-${builtins.replaceStrings [ "." ] [ "-" ] version}";
    sha256 = "sha256-SQH85FLfVEEtYhR1+A1XxCDMiTjDgLQX6zifbLxCa5c=";
  };

  nativeBuildInputs = [ autoreconfHook ];
```